src/tests/test_cloudBuffer_influxWrapper.py

```diff
@@ -1,9 +1,85 @@
 import unittest
 
+from cloudBuffer.influxWrapper import InfluxWrapper
 
-class MyTestCase(unittest.TestCase):
-    def test_something(self):
-        self.assertEqual(True, False)
+
+class InfluxWrapperTest(unittest.TestCase):
+
+    def setUp(self):
+        """
+        setUp method
+        """
+        self.valid_connection_params = {
+            "host": "localhost",
+            "organization": "TUD",
+            "bucket": "A",
+            "token": "my_token"
+        }
+
+        self.influxWrapper = InfluxWrapper(self.valid_connection_params)
+
+    def test_connection_params(self):
```

Comment:
I think we should be testing valid and invalid connection params in different methods.
Testing actual functionality of this wrapper would be at least a nice to have, but I acknowledge how time consuming that task might be.